src/pages/SendBulkEmail.page

```diff
@@ -163,7 +163,7 @@
                     <apex:outputLabel value="{!$Label.labelMessageBody}" styleClass="slds-form-element__label" for="readMessage"/>
                     <div class="slds-form-element__control">
                         <div class="slds-text-body--regular">
-                            <apex:outputText id="txtMessage" value="{!strMessage}" escape="false" />
+                            <apex:outputText id="txtMessage" value="{!HTMLENCODE(strMessage)}" escape="false" />
```

Comment:
No it is not safe!  i messed up the formatted display of the template, so remove the HTMLENCODE.  thanks.